mcs/class/System.Data/Makefile

```diff
@@ -4,7 +4,7 @@ include ../../build/rules.make
 
 LIBRARY = System.Data.dll
 
-LIB_REFS = System System.Xml System.Core System.Numerics System.Transactions
+LIB_REFS = System System.Xml System.Core System.Numerics System.Transactions System.Drawing
```

Comment:
Do we need `System.Drawing` here? As I recall this is a library we specifically omit from our profile.

Reply:
I see System.Drawing in our aot and jit directories of the install.  It should be fine